sdk/storage/azure-storage-blobs/CMakeLists.txt

```diff
@@ -52,12 +52,13 @@ set(
     inc/azure/storage/blobs/dll_import_export.hpp
     inc/azure/storage/blobs/page_blob_client.hpp
     inc/azure/storage/blobs/rest_client.hpp
+    inc/azure/storage/blobs/rtti.hpp
     inc/azure/storage/blobs.hpp
-    src/private/package_version.hpp
 )
 
 set(
   AZURE_STORAGE_BLOBS_SOURCE
+    src/private/package_version.hpp
```

Comment:
We keep the files in alphabetical order in storage.

Reply:
This is a header, why is it put in SOURCE ?

Reply:
it is in `src` folder, i.e. not something discoverable by the code that has path to include directory. I don't think either thing makes a technical difference, but right now, it was useful for visual comparison against Solution Explorer.